core/sdk.go

```diff
@@ -128,13 +251,13 @@ func NewSDK(
 	}
 
 	return rondImpl{
-		evaluator:        evaluator,
-		oasRouter:        oasRouter,
-		evaluatorOptions: evaluatorOptions,
-		oas:              oas,
+		partialResultEvaluators: evaluator,
+		oasRouter:               oasRouter,
+		evaluatorOptions:        evaluatorOptions,
+		oas:                     oas,
+		opaModuleConfig:         opaModuleConfig,
 
-		metrics:  m,
-		registry: registry,
```

Comment:
you are not using registry anymore but it is still provided as an argument, can you remove it or there will be need for it in the future?

Reply:
It is used in the NewSDK function